docs/metrics/customizing-the-sdk/README.md

```diff
@@ -235,8 +235,7 @@ with the metric are of interest to you.
 #### Specify custom boundaries for Histogram
 
 By default, the boundaries used for a Histogram are [`{ 0, 5, 10, 25, 50, 75,
-100, 250, 500,
-1000}`](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/metrics/sdk.md#explicit-bucket-histogram-aggregation).
+100, 250, 500,750, 1000, 2500, 5000, 7500, 10000}`](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/metrics/sdk.md#explicit-bucket-histogram-aggregation).
```

Comment:
```suggestion
100, 250, 500,750, 1000, 2500, 5000, 7500, 10000}`](https://github.com/open-telemetry/opentelemetry-specification/blob/v1.13.0/specification/metrics/sdk.md#explicit-bucket-histogram-aggregation).
```

Reply:
Good point! But since it's currently an unreleased change (https://github.com/open-telemetry/opentelemetry-specification/blob/main/CHANGELOG.md#unreleased), should we wait with merging this until it's part of the next spec version?

Reply:
Fortunately, we do not have to wait. It was released as 1.14. My previous comment is updated.